ci/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:20.04
+RUN apt-get update -qq && \
+    apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    gcc \
+    libc6-dev
+
+RUN curl --proto '=https' --tlsv1.2 -sSf -o soteria-linux-rust.tar.gz https://supercompiler.xyz/y/soteria-linux-rust.tar.gz
+RUN tar xf soteria-linux-rust.tar.gz
+ENV SOTERIA_LICENSE_KEY=2zCz2GgSoSS68eNJENWrYB48dMM1zmH8SZkgYneVDv2G4gRsVfwu5rNXtK5BKFxn7fSqX9BvrBc1rdPAeBEcD6Es
```

Comment:
`SOTERIA_LICENSE_KEY` - 😬 

Using proprietary tools is another strike in my books. This basically amounts to an advertisement.